### DISCUSSION TITLE
煎饼摊

### DISCUSSION THREAD

**Discussion by ShevonKuan** [chinese-poetry/chinese-poetry#353] [Ideas]
最近发现一个很好玩的小游戏叫做煎饼摊，一种飞花令与英文填词游戏的结合。我借用了贵项目的数据，十分感谢
[项目地址](https://github.com/ShevonKuan/pancake-stall)
效果是这样的。
![](https://github.com/ShevonKuan/pancake-stall/raw/main/demo/Demo_20x50_2.jpg)

**Comment by outbo** [comment]
小博已经收到您发的邮件
